scripts/image-builder/configure.sh

```diff
@@ -31,21 +34,20 @@ else
     composer_packages="osbuild-composer osbuild"
 fi
 
-# shellcheck disable=SC2086
-sudo dnf install -y \
-     ${composer_packages} \
+"${DNF_RETRY}" install \
+     "${composer_packages} \
      git composer-cli ostree rpm-ostree \
      cockpit-composer bash-completion podman runc genisoimage \
      createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
-     containernetworking-plugins expect
+     containernetworking-plugins expect"
 
 sudo systemctl enable osbuild-composer.socket --now
 sudo systemctl enable cockpit.socket --now
 sudo firewall-cmd --add-service=cockpit --permanent
 
 # The mock utility comes from the EPEL repository
-sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OSVERSION}.noarch.rpm"
-sudo dnf install -y mock nginx tomcli parallel
+"${DNF_RETRY}" install "https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OSVERSION}.noarch.rpm"
```

Comment:
for readability i would also put the 2nd parameter  in  apostrophes "${DNF_RETRY}" "install"

Reply:
Done